examples/bri-3/prisma/schema.prisma

```diff
@@ -0,0 +1,19 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = env("DATABASE_URL")
+}
+
+model BpiSubject {
```

Comment:
Just to understand this better, we have the similar declaration under migrations also. Does this model not define the table by default?
Also, are we going to do this pattern, I mean one migration file declaration and one in the model here for other models?

Reply:
This is just how prisma works. We only define (and change) our persistence models in schema.prisma, and then generate a migration and a prisma client out if it by using Prisma commands. I ll add a readme with the process in subsequent PRs. 

What is good about it is that the persistence model is completely separated from our domain objects, and we just map in both directions on the level of repositories.